specs/user_spec.rb

```diff
@@ -59,4 +58,62 @@
       end
     end
   end
+
+
+  describe "user#net_expenditures" do
+    before do
+      @user = RideShare::User.new(id: 9, name: "Merl Glover III",
+                                  phone: "1-602-620-2330 x3723", trips: [])
```

Comment:
I think you're missing some edge cases that are common to both these `User` methods:

- What happens if the user has no trips?
- What happens if the user has an incomplete trip?